docs/governance/templates/release_issue.md

```diff
@@ -63,7 +63,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Post to the [agonesdev](https://twitter.com/agonesdev) Twitter account.
 - [ ] Run `git checkout main`.
 - [ ] Run `make sdk-update-version release_stage=after version={version}` file. This command will update the version number in the sdks/install files to {version}+1-dev.
-- [ ] Update the `agones` crate version in `sdks/rust/Cargo.toml` to {version}, then run `cargo publish --dry-run` and, if successful, proceed with `cargo publish`
+- [ ] Update the `agones` crate version in `sdks/rust/Cargo.toml` to {version}. First, run `cargo login` to authenticate with `crates.io`. Afterward, execute `cargo publish --dry-run`. If that is successful, proceed with `cargo publish`.
```

Comment:
Optional: 

https://github.com/googleforgames/agones/blob/97057bf8a2eaa1cd4a3a0ca05c8bde870e320263/build/includes/sdk.mk#L201-L217

Could also make a `make sdk-shell-rust` target and use that command line to do the publishing of the Rust crate (rather than install tools locally).

Not sure if that's how you are doing the node deployments?

Reply:
Added `sdk-shell-rust` target but how can we test it? If we execute this target, will it be added in the Agones crate again?